model/activist.go

```diff
@@ -114,45 +150,64 @@ func GetActivistJSON(db *sqlx.DB, options GetActivistOptions) (ActivistJSON, err
 }
 
 func getActivistsJSON(db *sqlx.DB, options GetActivistOptions) ([]ActivistJSON, error) {
```

Comment:
Is this function used anywhere? If not, let's delete it

Reply:
Indeed it is